config.json

```diff
@@ -422,6 +422,18 @@
       "unlocked_by": "hello-world",
       "uuid": "c3e89c7c-3a8a-4ddc-b653-9b0ff9e1d7d8"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "variable-length-quantity",
+      "topics": [
+        "lists",
```

Comment:
I like these topics!

Any others? The list can be found [here](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt)

Reply:
How about conditionals and loops? And maybe exception handling once I refactor the error cases. Any other suggestions?

Reply:
I think those sound sensible - `exception-handling` especially if we refactor the tests! :slightly_smiling_face: